### ISSUE TITLE
Add nginx to docker-compose

### ISSUE THREAD

**Issue by randomorder** [geosolutions-it/djam#41]
It would be nice to have an nginx added to the compose file to allow the users to spin up djam in production mode (development is using django server by default),

**Comment by giohappy** [comment]
I agree @randomorder. The reason it's not there right now is that DJAM until now has been deployed inside other compose, never standalone.
We could create a new compose file that includes the basic one.
